serverless/pages/custom-roles.asciidoc

```diff
@@ -13,7 +13,7 @@ preview:[]
 
 This content applies to: {es-badge} {sec-badge}
 
-The built-in <<general-assign-user-roles,organization-level roles>> and <<general-assign-user-roles,instance access roles>> are great for getting started with {serverless-full}, and for system administrators who do not need more restrictive access.
+The built-in <<general-assign-user-roles-organization-level-roles,organization-level roles>> and <<general-assign-user-roles-instance-access-roles,instance access roles>> are great for getting started with {serverless-full}, and for system administrators who do not need more restrictive access.
 
 As an administrator, however, you have the ability to create your own roles to describe exactly the kind of access your users should have within a specific project.
```

Comment:
"your own roles" is a little confusing given that we just talked about how they can use the built in role themselves. Should be something like you can create roles for users with the access they need within specific projects.

Reply:
corrected this text. This was confusing.